File: 7/Main.java
class Main {
  public static void main(String[] args) {
    int lastPrime = 13;
    int i = 6;
    while (i<10_001) {
      lastPrime +=2;
      boolean isPrime = true;
      for (int j=2; j<=Math.sqrt(lastPrime); j++) {
        if (lastPrime % j ==0) {
          isPrime = false;
          break;
        }
      }
      if (isPrime) i++;
    }
    System.out.println(lastPrime);
  }
}

File: 7/Main.go
package main

import (
	"fmt"
	"math"
)

func main() {
	lastPrime := 13
	i := 6
	for i < 10001 {
		lastPrime += 2
		isPrime := true
		for j := 2; j <= int(math.Sqrt(float64(lastPrime))); j++ {
			if lastPrime%j == 0 {
				isPrime = false
				break
			}
		}
		if isPrime {
			i++
		}
	}
	fmt.Println(lastPrime)
}
